notification/app/notification/NotificationApplicationLoader.scala

```diff
@@ -22,7 +25,16 @@ import scala.concurrent.ExecutionContext
 class NotificationApplicationLoader extends ApplicationLoader {
   override def load(context: Context): Application = {
     LoggerConfigurator(context.environment.classLoader) foreach { _.configure(context.environment) }
-    (new BuiltInComponentsFromContext(context) with AppComponents).application
+    val identity = AppIdentity.whoAmI(defaultAppName = "notification", defaultStackName = "mobile-notification")
+    val config = ConfigurationLoader.load(identity) {
+      case AppIdentity(app, stack, stage, _) => S3ConfigurationLocation (
```

Comment:
I just remembered the conversation we were having about it. I was wrong:
Configured like that, the loader will always go to s3, since the first case will always match, even if stage == "DEV"

Alternatively you could define it like that:
`case AppIdentity(app, stack, "CODE" | "PROD", _)`

So the DEV case falls back on the default. Maybe this is a sign I should formulate the library differently? Happy to hear your about on it.

Reply:
No, that's my bad - I should have been clearer about what I wanted. The solution you propose doesn't quite work though. There needs to be some way if capturing the value for stage. Does 

`case AppIdentity(app, stack, stage) if List("CODE","PROD").contains(stage) =>` . 

Does that work?

Reply:
You're right that's better, maybe I'd go one step further and use `if (stage != "DEV")`

Reply:
Maybe I could enhance the configuration library by providing two different types depending on if it's a live environment or a test one.

Reply:
I'll fix it as suggested and then if you've time to upgrade the library, I can make a change?

Reply:
oh yeah these are two separate problems. Just throwing ideas out there.